packages/auth/package.json

```diff
@@ -61,6 +70,10 @@
         "require": "./dist/browser-cjs/internal.js",
         "import": "./dist/esm2017/internal.js"
       },
+      "extension": {
```

Comment:
Need types, import, require, default (see previous comment). For types just use extension-cjs/internal/index.d.ts

Reply:
Done. Could you see if the import and default fields are correct.
Not sure if using "internal.js" is correct.

Reply:
Looks correct!